kovri.md

```diff
@@ -1,64 +1,68 @@
+A lightweight and security-focused router, Kovri is fully compatible with the I2P network.
 
-Since I2P routers know little or no information about the data they are sending, where it originated, or its destination, all information in this network is private and anonymous.
-If Alice sends her transaction with Kovri, nodes that log Monero traffic will no longer get her real IP address, rendering these passive surveillance tactics useless.
+If Alice sends her transaction with Kovri, nodes that log Monero traffic will no longer see her internet IP address, rendering passive surveillance tactics close to useless.
 
 Now let’s imagine a different scenario.
-Suppose Alice has an internet service provider that does not want her to use Monero, who send her a warning saying so.
-Alice can optionally send all the Monero sync data, not just the transaction broadcast, through I2P.
+Suppose Alice has an internet service provider that does not want her to use Monero, and send her a warning to cease and desist.
+Alice can optionally send all the Monero sync data, not just the transaction broadcast, through Kovri.
 Now, her Monero connections are hidden from her ISP.
 This improves Monero's censorship resistance, since other people no longer know who is running a Monero node and demand that they be shut down.
 
 Let’s look at one last scenario.
-Suppose George is the operator of a mining pool that controversially donates a portion of the block reward to a political party.
+Suppose Charlie is the operator of a mining pool that controversially donates a portion of the block reward to a political party.
 Other nodes could purposefully reject his solved blocks to express their disagreement with this policy.
 This sort of attack is called a Sybil attack.
-However, if George can send his solved blocks through Kovri, then no one can censor the work from his pool, since they no longer know that his pool created it.
+However, if Charlie can send his solved blocks through Kovri, then no one can censor the work from his pool, since they no longer know that his pool created it.
 
-Kovri will come bundled with future Monero releases, and it will be enabled by default to broadcast the transaction through the I2P network.
-It will feature a common API that can be used by any cryptocurrency, not just Monero.
+Kovri will come bundled with future Monero releases, and it will be enabled by default to broadcast the transaction anonymously.
+Kovri will feature a common API that can be used by any cryptocurrency, not just Monero.
 
-In summary, Kovri is a software router that allows Monero users to connect to the private I2P network to send transactions anonymously.
-Malicious nodes can no longer threaten users who create these transactions or block them from propagating across the network. 
+In summary, Kovri is an anonymizing router software that allows Monero users to send transactions anonymously while enjoying the benefits of the I2P network.
```

Comment:
"In summary, Kovri is an anonymizing router software that allows Monero users to send transactions anonymously with less privacy and security concerns."

Reply:
Meh, I don't care for this revision. Let's come up with a 3rd that's better than our previous two.